params.go

```diff
@@ -246,6 +251,29 @@ type ParamsContainer interface {
 	GetParams() *Params
 }
 
+type RawParams struct {
+	Params        `form:"*"`
+	APIMode       APIMode `form:"-"`
+	StripeContext string  `form:"-"`
+}
+
+func (p *RawParams) GetAPIMode() APIMode {
```

Comment:
Do we need these functions considering fields are public?

Reply:
Good point, we don't